Sources/TokamakCore/App/Scenes/WindowGroup.swift

```diff
@@ -62,7 +62,8 @@ public struct WindowGroup<Content>: Scene, TitledScene where Content: View {
     self.title = Text(title)
     self.content = content()
   }
-
+    
+  @_spi(TokamakCore)
```

Comment:
Could you apply SwiftFormat as `swiftformat .` in the project directory to get rid of trailing whitespaces? You can also install a pre-commit hook [as described here](https://github.com/TokamakUI/Tokamak#coding-style), which will apply formatting fixes automatically for you before you commit.

Reply:
Thanks, I had forgotten that.